src/v8/v8.cc

```diff
@@ -298,67 +281,33 @@ bool V8::load(const std::string &code, bool allow_precompiled) {
   }
 
   if (!module_) {
-    const auto stripped_source = getStrippedSource();
-    module_ = wasm::Module::make(store_.get(), stripped_source ? stripped_source : source_);
+    std::string stripped;
+    if (!common::WasmUtil::getStrippedSource(code, stripped)) {
+      fail(FailState::UnableToInitializeCode, "Failed to parse corrupted Wasm module");
+      return false;
+    };
+    wasm::vec<byte_t> code_vec = wasm::vec<byte_t>::invalid();
+    if (stripped.empty()) {
+      // Use the original bytecode.
+      code_vec = wasm::vec<byte_t>::make(code.size(), (char *)(code.data()));
```

Comment:
`(char *)(code.data())` is a temporary workaround for now to pass build and we should fix. This is necessary since wasm::vec<byte_t>::makerequires us to pass char vector instead of const char vector. If we want to do zero copy, we need to do cast like this or change the arg type of `code` to `std::string&`.. but I'm not sure why ::make require non-const bytes code since V8 seems to copy the code internally anyway..

Reply:
@PiotrSikora I need your inputs here.. Should we change the signature of `load` to `std::string &code` from `const std::string &code` so that we can have `vector<char>` instead of `vector<const char>` from `code.data()`, or we allow `(char *)(code.data())`..?

Reply:
The following is the error we get without modifying signature and just pass `code.data()`
```
external/com_googlesource_chromium_v8/wee8/third_party/wasm-api/wasm.hh:149:18: error: cannot initialize an array element of type 'char' with an rvalue of type 'typename std::remove_reference<const char *&>::type' (aka 'const char *')
    T data[] = { std::move(args)... };
                 ^~~~~~~~~~~~~~~
external/proxy_wasm_cpp_host/src/v8/v8.cc:293:37: note: in instantiation of function template specialization 'wasm::vec<char>::make<unsigned long, const char *>' requested here
      code_vec = wasm::vec<byte_t>::make(code.size(), code.data());
```

Reply:
I think this might be the reason why we originally copied the bytecode. Perhaps make `getStrippedSource` always return `std::string` (either stripped version or a copy of the original bytecode) and then use it as input here?

Reply:
so we cannot avoid copying the original bytecode anyway.. thanks for suggestion.

Reply:
done https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/152/commits/ed0f3a5b02d48e5412f7b55033e4f6bed142a16d

Reply:
The bytecode read from the configuration is passed to both: canary and base WasmVMs at startup, so I'm afraid that we have to copy it, unfortunately.

Perhaps we can rewrite this logic somehow, but that should probably be done in a separate PR.